Slim/Menu/BrowseLibrary/Releases.pm

```diff
@@ -185,6 +188,23 @@ sub _releases {
 		push @items, _createItem($name || ucfirst($role), [ { searchTags => [@$searchTags, "role_id:$role", "album_id:" . join(',', @{$contributions{$role}})] } ]);
 	}
 
+	# Add item for Classical Works if the artist has any.
```

Comment:
Didn't we say that all of this works treatment should remain optional?...

Reply:
No, composer would only be put in the album title if $_->{composer} exists, and it won't unless we've come from Works.pm, and in that case it needs to be there.

On the "optional" point, I'll respond to this in a reply to your summary post above (which will appear at the bottom after your "changelog" comment)